{"name": "test_001_login_success[test1234-test1234-\\u767b\\u5f55\\u6210\\u529f1]", "status": "broken", "statusDetails": {"message": "AttributeError: 'NoneType' object has no attribute 'get'", "trace": "self = <testcase.test01_login.TestLogin object at 0x1027e6550>\nuname = 'test1234', upwd = 'test1234', expect = '登录成功'\n\n    @pytest.mark.parametrize(\"uname, upwd, expect\", utils.get_login_data_by_excel(2, 3))\n    # 定义日志级别为CRITICAL\n    @allure.severity(allure.severity_level.CRITICAL)\n    def test_001_login_success(self, uname, upwd, expect):\n        logging.info(\"用例输入数据如下：用户名：{}，密码：{}，预期结果：{}\".format(uname, upwd, expect))\n        # 发送请求附带数据\n        json_data = {\"uname\": uname, \"upwd\": upwd}\n        response = self.login_api.login(json_data)\n        # 打印获取到的token数据\n>       print(response.json().get(\"content\").get(\"token\"))\nE       AttributeError: 'NoneType' object has no attribute 'get'\n\ntest01_login.py:31: AttributeError"}, "parameters": [{"name": "uname", "value": "'test1234'"}, {"name": "upwd", "value": "'test1234'"}, {"name": "expect", "value": "'登录成功'"}], "start": 1710170653999, "stop": 1710170654579, "uuid": "9d2403eb-6079-43e9-a978-8619d5d51e61", "historyId": "6261ef6205cdb334abd6178168c8e559", "testCaseId": "5e617d0a494d476c894db1d54060c1ec", "fullName": "testcase.test01_login.TestLogin#test_001_login_success", "labels": [{"name": "severity", "value": "critical"}, {"name": "parentSuite", "value": "testcase"}, {"name": "suite", "value": "test01_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "hkdeMacBook-Pro"}, {"name": "thread", "value": "65215-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcase.test01_login"}]}